Add WebOps announcement (#1144)
vtex/faststore: docs/blog/2022-02-08-webops.md

---
title: Improved WebOps pipeline execution time
description: Improvements in WebOps pipeline execution time.
tags: [webops]
hide_table_of_contents: false
---

> Hello WebOps users,
> 
> With the increased number of concurrent users of the WebOps platform, we noticed WebOps pipelines were taking longer to execute. 
> Under high loads, workflows that used to take from 2 to 10 minutes started to span between 20 and 50 minutes. **This caused CMS publications to take longer than they should, harming the developer experience.**
> 
> Earlier this week, we began an investigation into it, and we managed to improve the situation drastically. **Our pipelines runtime is now back to the 2-10 minutes range, depending on the workload type and size of the store.**
> 
> That's it for this announcement, but expect further improvements during the closed beta as we work on our platform's stability and performance!

Author: ```suggestion
Hello WebOps users,

With the increased number of concurrent users of the WebOps platform, we noticed WebOps pipelines were taking longer to execute. 
Under high loads, workflows that used to take from 2 to 10 minutes started to span between 20 and 50 minutes. **This caused CMS publications to take longer than they should, harming the developer experience.**

Earlier this week, we began an investigation into it, and we managed to improve the situation drastically. **Our pipelines runtime is now back to the 2-10 minutes range, depending on the workload type and size of the store.**

We are working hard to deliver improvements to our platform such as this. Expect further improvements during the closed beta as we work on our platform's stability and performance!
```

> title: Improved WebOps pipeline execution time
> description: Improvements in WebOps pipeline execution time.
> tags: [webops]
> hide_table_of_contents: false
> ---
> 
> Hello WebOps users,
> 
> With the increased number of concurrent users of the WebOps platform, we noticed WebOps pipelines were taking longer to execute. 
> Under high loads, workflows that used to take from 2 to 10 minutes started to span between 20 and 50 minutes. **This caused CMS publications to take longer than they should, harming the developer experience.**
> 
> Earlier this week, we began an investigation into it, and we managed to improve the situation drastically. **Our pipelines runtime is now back to the 2-10 minutes range, depending on the workload type and size of the store.**
> 
> That's it for this announcement, but expect further improvements during the closed beta as we work on our platform's stability and performance!

Author: ```suggestion
title: Improved WebOps build time
description: Improvements in WebOps build time.
tags: [webops]
hide_table_of_contents: false
---

Hello WebOps users,

With the increased number of concurrent users of the WebOps platform, we noticed WebOps pipelines were taking longer to execute. 
Under high loads, workflows that used to take from 2 to 10 minutes started to span between 20 and 50 minutes. **This caused CMS publications to take longer than they should, harming the developer experience.**

Earlier this week, we began an investigation into it, and we managed to improve the situation drastically. **Our pipelines runtime is now back to the 2-10 minutes range, depending on the workload type and size of the store.**

We are working hard to deliver improvements to our platform such as this. Expect further improvements during the closed beta as we work on our platform's stability and performance!
```
